### PR TITLE
Update `Environment Variables` section in docs

### PR DESCRIPTION
**Pull request by bryannho**
## Changes

- Rename `env-vars.md` to `secrets.md` and replace all occurences of `environment variables` to `secrets`
- Update `Environment variables` section in `cli.md`
- Replace static images

Addresses #169 

----
📚 Documentation preview 📚: https://ploomber-doc--171.org.readthedocs.build/en/171/

### CODE REVIEW

File: doc/user-guide/cli.md [hunk 1]
```diff
@@ -94,11 +94,11 @@ ploomber-cloud examples flask/basic-app
...
 A full list of example applications is available [here](https://github.com/bryannho/doc/tree/main/examples)
 
-## Defining environment variables
+## Defining secrets
 
-If your project uses environment variables, you can define them in an `.env` file.
+If your project uses secrets, you can define them in an `.env` file.
 
-In your main project directory, create an `.env` file. Open it in your code editor, and enter your environment variables. It should look like this:
+In your main project directory, create an `.env` file. Open it in your code editor, and enter your secrets. It should look like this:
```
Comment (neelasha23) [inline doc/user-guide/cli.md:101]: for consistency `MY_ENV_VAR_1` and `MY_ENV_VAR_2` can be renamed to `MY_SECRET_1` and `MY_SECRET_2`

File: doc/user-guide/secrets.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+value = environ.get("MY_SECRET")
+
+if value is None:
+    print("MY_SECRET is undefined!")
+else:
+    print(f"MY_SECRET: {value}")
+```
+
+
+A complete example is available [here.](https://github.com/ploomber/doc/blob/main/examples/voila/env-variable/app.ipynb)
```
Comment (neelasha23) [inline doc/user-guide/secrets.md:43]: Do we need to rename this project also to `secrets`? And maybe change `value = environ.get("ENV_VAR", None)` to `value = environ.get("SECRET", None)`

File: doc/user-guide/cli.md (hunk 1 above)
Comment (edublancas) [reply doc/user-guide/cli.md:101]: I agree

File: doc/user-guide/secrets.md (hunk 2 above)
Comment (edublancas) [reply doc/user-guide/secrets.md:43]: agree

**Review by edublancas — CHANGES_REQUESTED** [review]

File: doc/user-guide/secrets.md (hunk 2 above)
Comment (bryannho) [reply doc/user-guide/secrets.md]: [renamed the project and fixed the link](https://github.com/ploomber/doc/pull/171/files#diff-8f7b69d5cb7da4e9386d20f6b5c3645123a4d09eacba1fd8d168e164e0cdaf8cR43)

**Pull request merged**